ISISScientificComputing/autoreduce-frontend
# 1423 keep pagination ar 1423

### Summary of work
Maintains URL queries after clicking on a run in the instrument runs page and after navigating back to the runs page by clicking the 'Back to <InstrumentName> runs' button.

### How to test your work
1. Checkout this branch and run the web app.
2. Navigate to http://127.0.0.1:8000/runs/LARMOR/
3. Change 'Items per page' to 50.
4. Navigate to the second page of runs.
5. Click the top run.
6. `?page=2&pagination=50&sort=run` should be visible in the URL.
7. Click the 'Back to LARMOUR runs' button.
8. The URL should still contain `?page=2&pagination=50&sort=run` and you should still be on the second page of runs with 50 runs listed.
9. Stop the web app and run the tests at `autoreduce_frontend/selenium_tests/tests/pages/test_runs_list_page.py`.

Fixes AR-1423

**Before merging ensure the release notes have been updated**
